### PR TITLE
board/imxrt1020-evk: Add Quad Timer support for imxrt1020

### PR DESCRIPTION
**Pull request by yashwanth686007**
This patch adds support of quad timer on imxrt1020-evk
chipset

Signed-off-by: Yashwanth <v.yashwanth@samsung.com>

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [353cbe589b93e53f8f652e9d2a7a6809c6992505] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [353cbe589b93e53f8f652e9d2a7a6809c6992505] - Code Rule Check OK.

File: os/arch/arm/src/imxrt/imxrt_qtmr.c [hunk 1]
```diff
@@ -77,6 +77,7 @@ static struct imxrt_qtmr_chipinfo_s imxrt_qtmr1_chipinfo = {
 
 };
 
+#ifdef CONFIG_ARCH_CHIP_FAMILY_IMXRT105x
```
Comment (sunghan-chang) [inline os/arch/arm/src/imxrt/imxrt_qtmr.c]: Why is this necessary? Doesn't 1020 have 4 channels?

Comment (yashwanth686007) [reply os/arch/arm/src/imxrt/imxrt_qtmr.c]: 1020 has only two qtimers available.

Comment (sunghan-chang) [reply os/arch/arm/src/imxrt/imxrt_qtmr.c]: Could you let me know where did you find that? When I checked docs and source codes, it has 4 for 1020 as well.
IMXRT1020RM.pdf, page 2482
```
48.3 Features
The TMR module design includes these distinctive features:
• Four 16-bit counters/timers
```

Comment (sunghan-chang) [reply os/arch/arm/src/imxrt/imxrt_qtmr.c]: Ok. I see. I found there is no QTIMER3 and 4 in imxrt102x_config.h
